assembl/views/search/views.py

```diff
@@ -0,0 +1,62 @@
+import json
+from pyramid.events import NewRequest
+from pyramid.view import view_config
+
+from assembl.indexing.utils import connect
+from assembl.indexing.settings import get_index_settings
+from assembl.indexing.changes import changes, in_tests
+from assembl import models
+from assembl.lib.sqla import get_session_maker
+
+def get_curl_query(query):
+    return "curl -XGET 'localhost:9200/_search?pretty' -d '{}'".format(
+        json.dumps(query).replace("'", "\\u0027"))
+
+
+@view_config(route_name='search', renderer='json')
+def search_endpoint(context, request):
+    es = connect()
+    index_name = get_index_settings()['index_name']
+    query = request.json_body
+#    print get_curl_query(query)
+    result = es.search(index=index_name, body=query)
+
+    # add creator_name in each hit
+    creator_ids = set([hit['_source']['creator_id']
+                       for hit in result['hits']['hits']
+                       if hit['_source'].get('creator_id', None) is not None])
+    session = get_session_maker()
+    creators = session.query(models.AgentProfile.id, models.AgentProfile.name
+        ).filter(models.AgentProfile.id.in_(creator_ids)).all()
+    creators_by_id = dict(creators)
+    # u'query': {u'bool': {u'filter': [{u'term': {u'discussion_id': u'23'}}]}}
+    filters = [fil for fil in query['query']['bool']['filter']]
+    discussion_id = [f.values()[0].values()[0]
```

Comment:
ok for your fix https://github.com/assembl/assembl/pull/84/commits/a5126bb1356e086a72e5fdfadd05aade441f64a5